test/kubectl-asset.integ.ts

```diff
@@ -0,0 +1,42 @@
+import * as path from 'path';
+import * as cdk from 'aws-cdk-lib';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as cr from 'aws-cdk-lib/custom-resources';
+
+import { KubectlAsset } from '../lib';
+
+/**
+ * Test verifies that AWS CLI is invoked successfully inside Lambda runtime.
+ */
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'lambda-layer-kubectl-integ-stack');
+const asset = new KubectlAsset(stack, 'kubectl-asset');
+const layer = new lambda.LayerVersion(stack, 'KubectlLayer', {
+  code: lambda.Code.fromBucket(asset.bucket, asset.s3ObjectKey),
+  description: '/opt/kubectl/kubectl and /opt/helm/helm',
+});
+
+const runtimes = [
+  lambda.Runtime.PYTHON_3_7,
+  lambda.Runtime.PYTHON_3_9,
+];
```

Comment:
There's no restriction that kubectl has to be used with Python runtimes but to test other languages I'd need to write different lambda handlers. i feel like its not worth it and that this shows enough.